.circleci/config.yml

```diff
@@ -52,8 +52,6 @@ jobs:
           path: "Angular InstantSearch/getting-started"
       - build_project:
           path: "Angular InstantSearch/infinite-scroll"
-      - build_project:
-          path: "Angular InstantSearch/loading-indicator"
```

Comment:
already in https://github.com/algolia/doc-code-samples/pull/261/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R40